### PR TITLE
feat: Upgrade to Mender version `3.6.3`.

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: None
Ticket: None

### CODE REVIEW

File: other-components.yml [hunk 1]
```diff
@@ -5,16 +5,16 @@
 # 'services' tag.
 services:
     mender-artifact:
-        image: mendersoftware/mender-artifact:3.10.1
+        image: mendersoftware/mender-artifact:3.10.2
 
     mender-cli:
-        image: mendersoftware/mender-cli:1.11.0
+        image: mendersoftware/mender-cli:1.11.1
```
Comment (merlin-northern) [inline other-components.yml:11]: I cant pull this image:
```bash
# docker pull mendersoftware/mender-cli:1.11.1
Error response from daemon: pull access denied for mendersoftware/mender-cli, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
should it be like that?

File: other-components.yml [hunk 2]
```diff
@@ -5,16 +5,16 @@
 # 'services' tag.
 services:
     mender-artifact:
-        image: mendersoftware/mender-artifact:3.10.1
+        image: mendersoftware/mender-artifact:3.10.2
```
Comment (merlin-northern) [inline other-components.yml:8]: I cannot pull this image:
```bash
# docker pull mendersoftware/mender-artifact:3.10.2
Error response from daemon: pull access denied for mendersoftware/mender-artifact, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
should it be like that?

File: git-versions.yml [hunk 3]
```diff
@@ -50,7 +50,7 @@ services:
         image: mendersoftware/mender:3.5.1
 
     mender-connect:
-        image: mendersoftware/mender-connect:2.1.0
+        image: mendersoftware/mender-connect:2.1.1
```
Comment (merlin-northern) [inline git-versions.yml:53]: I cannot pull this image:
```bash
# docker pull mendersoftware/mender-connect:2.1.1
Error response from daemon: pull access denied for mendersoftware/mender-connect, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
should it be like that?

**Review by merlin-northern — COMMENTED** [review]
as a review I checked if I can pull the images. I cant pull all of them, I wonder should it be like that? since I cant pull the old tags, I believe that we are just not pushing those components. just reporting.

File: other-components.yml (hunk 1 above)
Comment (tranchitella) [reply other-components.yml:11]: These are not Docker images (I know, I know).
It is just a way to express verisons.

File: other-components.yml (hunk 2 above)
Comment (tranchitella) [reply other-components.yml:8]: These are not Docker images (I know, I know).
It is just a way to express versions.

File: git-versions.yml (hunk 3 above)
Comment (tranchitella) [reply git-versions.yml:53]: These are not Docker images (I know, I know).
It is just a way to express versions.

**Review by merlin-northern — APPROVED** [review]
thanks for answers

**Comment by tranchitella** [comment]
I won't run the QA pipeline as the change doesn't affect it.

**Pull request merged**